h2o-r/tests/testdir_algos/glrm/runit_PUBDEV_7609_glrm_representation_loading_name.R

```diff
@@ -0,0 +1,30 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../../scripts/h2o-r-test-setup.R")
+
+test.glrm.arrests <- function() {
+  Log.info("Importing arrests.csv data...") 
+  arrestsR <- read.csv(locate("smalldata/pca_test/USArrests.csv"), header = TRUE)
+  arrestsH2O <- h2o.uploadFile(locate("smalldata/pca_test/USArrests.csv"), destination_frame = "arrestsH2O")
+  initCent <- scale(arrestsR, center = TRUE, scale = FALSE)[1:4,]
+  loading_name = "are_you_confused"
```

Comment:
it is a confusing world!

however, I would still suggest to use

    loading_name <- "are_you_confused"

to make it less confusing ;)

Reply:
Right.  Let me change it!